webapp/utils/utils.jsx

```diff
@@ -737,19 +737,11 @@ export function resetTheme() {
     applyTheme(Constants.THEMES.default);
 }
 
-export function applyFont(fontName) {
+export function applyOpenSansFont() {
```

Comment:
We don't even need to add another class. You can just add `font-family: 'Open Sans', 'sans-serif';` to the `.app__body` class